certbot/certbot/_internal/cli/subparsers.py

```diff
@@ -32,8 +32,8 @@ def _create_subparsers(helpful):
                 " Currently --csr only works with the 'certonly' subcommand.")
     helpful.add("revoke",
                 "--reason", dest="reason",
-                choices=CaseInsensitiveList(sorted(constants.REVOCATION_REASONS,
-                                                   key=constants.REVOCATION_REASONS.get)),
+                choices=CaseInsensitiveList(
+                    dict(sorted(constants.REVOCATION_REASONS.items(), key=lambda item: item[1]))),
```

Comment:
Why are we converting this back into a dict? Don't we want the list of keys here?